docs/commands.md

```diff
@@ -187,6 +187,18 @@ Arguments:
 
 Result: Sends a message in the executed channel with content `message`. This message will be an embed if `embed` is true.
 
+### Edit Say Messages (MODS ONLY)
+Name: `edit-say-message`
+
+Arguments:
+* `messageToEdit` - The ID of the message contain the embed you'd like to edit - Snowflake
+* `newContent` - The new content for the message - Optional String
+* `newColor` - The new color for the embed - Optional Color (default: Blurple)
+* `channelOfMessage` - The channel the embed was originally sent in - Optional channel (default: Channel command was executed in)
+* `timestamp` - Whether to add the timestamp of when the message was originally sent or not - Defaulting Boolean (default: false)
```

Comment:
This should say optional boolean even though in the code it's a defaulting boolean. Or we should adjust the rest of the docs.